docs/source/en/using-diffusers/loading_adapters.md

```diff
@@ -157,6 +157,22 @@ image
 
 For both [`~loaders.LoraLoaderMixin.load_lora_weights`] and [`~loaders.UNet2DConditionLoadersMixin.load_attn_procs`], you can pass the `cross_attention_kwargs={"scale": 0.5}` parameter to adjust how much of the LoRA weights to use. A value of `0` is the same as only using the base model weights, and a value of `1` is equivalent to using the fully finetuned LoRA.
 
+For fine-grained control on how much of the LoRA weights are used, use [`~loaders.LoraLoaderMixin.set_adapters`]. Here, you can define scale of any granularity up to per-transformer.
```

Comment:
```suggestion
For more granular control over the amount of LoRA weights used per layer, you can use `scales` to define a dictionary specifying how much to scale the weights in each layer by.
```

Reply:
The example had an error. You use `pipe.set_adapters` (not `pipe.load_lora_weights`) to pass the `scales` dictionary.

Is this okay:
> For more granular control on the amount of LoRA weights used per layer, you can use [`~loaders.LoraLoaderMixin.set_adapters`] and pass a `scales` dictionary specifying how much to scale the weights in each layer by.

?

Reply:
My example had an error. You use `pipe.set_adapters` (not `pipe.load_lora_weights`) to pass the dictionary, and the dictionary param is called `adapter_weights`.

This is unfortunately a bit confusing, because `weights` also means the actual lora weights, and also the scale weights. But the naming was already here before this PR.

Does this work for you:

> For more granular control on the amount of LoRA weights used per layer, you can use [`~loaders.LoraLoaderMixin.set_adapters`] and pass a dictionary specifying how much to scale the weights in each layer by.

?

Reply:
Yeah that'll work, thanks for clarifying :)